packages/web/src/scss/components/Accordion/README.md

```diff
@@ -0,0 +1,47 @@
+# Accordion
+
+Accordion is a different wrapper for multiple **Collapse** items, and can only open one item.
```

Comment:
Please reword this sentence. The accordion has a lot of styles and is not only a wrapper.

Reply:
It will be fixed in the JS branch because it is a problem to maintain two versions of these files since `readme.md` and `index.html` are the same for both CSS and JS brances.

Reply:
Please fix this in this PR, because it is the main branch for this task. JS branch should solve a JS related things and this is problem related to this PR only.